### ISSUE TITLE
Illegible Text Combination For Bookmark List

### ISSUE THREAD

**Issue by Golutazem** [vaeth/bookmarkdupes#1]
The list of duplicated bookmarks is illegible as it is black text on a dark grey background. I guess the extension is specifying one color and using a default for the other.

**Comment by vaeth** [comment]
Thanks for the report.

> I guess the extension is specifying one color

No, the extension is not specifying any color (it does not even use a .css file).
So everything should be the default. Here, it is black on white. I do not know why it is different for you. Maybe you are using some theme/extension which specifies a very bad default?

**Comment by vaeth** [comment]
SInce bookmarkdupes-4.0 now uses a completely different mechanism (a tab instead of a panel), it might be that the problem is now resolved. Please open a new bug if a similar problem still occurs.

**Comment by Golutazem** [comment]
I have switched from Firefox to Pale Moon due to better bookmark support in Pale Moon. -- So I don't know when I will test the WebExtension. Pale Moon has no plan to support them. People caring much about managing bookmarks will use Pale Moon if they know about it.

**Comment by vaeth** [comment]
> I have switched from Firefox to Pale Moon

I had used Pale Moon for quite a while, but I no longer trust its security. They simply are not able to meet pulling from firefox upstream, and presumably hundreds of known vulnerabilities are still open. (E.g. even after a year or so, they _still_ have not fixed the assembler code relying on undocumented behaviour of gcc which causes crashes when compiled with gcc-6, and it still does not compile with gcc-7.)

> Pale Moon has no plan to support them

This was IMHO one of the main wrong strategical decisions. Although at a first glance it sounds nice that legacy extensions are supported, it means they would need even more man-power to pull from firefox upstream. Thus, even less vulnerabilities will be fixed, although this was already the main problem of palemoon.

Anyway, discussion about palemoon is off-topic here. Closing this bug for the time being.
